### DISCUSSION TITLE
Quarkus Rest Client base URI - 3 configuration properties

### DISCUSSION THREAD

**Discussion by akil-rails** [quarkusio/quarkus#45281] [Q&A] [answered]
The Quarkus REST client configuration gives 3 options to specify the base URI

- url
- uri
- override-uri

It's not very clear from the documentation , how these 3 are different . Can someone explain?

![image](https://github.com/user-attachments/assets/6f6c63b0-9071-4b07-85bf-1a61d7cf889a)

[a comment by an automated account is not shown]

**Comment by geoand** [comment, marked as answer]
> override-uri

Is not meant to used by applications. 

`uri` and `url` are effectively the same thing
